### PR TITLE
Add some log info to gain insight into some failure cases.

### PR DESCRIPTION
**Pull request by nilo19**
Add some log info to gain insight into some failure cases.

/sig testing

**Release note**:
```
None
```

### CODE REVIEW

File: tests/e2e/network/ensureloadbalancer.go [hunk 1]
```diff
@@ -144,6 +145,12 @@ var _ = FDescribe("Ensure LoadBalancer", func() {
 		}()
 		By("Waiting for exposure of internal service with specific IP")
 		err = utils.WaitUpdateServiceExposure(cs, ns.Name, serviceName, ip1, true)
+		list, errList := cs.CoreV1().Events(ns.Name).List(metav1.ListOptions{})
+		Expect(errList).NotTo(HaveOccurred())
+		By("Events list:")
+		for i, event := range list.Items {
+			By(fmt.Sprintf("%d. %v", i, event))
```
Comment (feiskyer) [inline tests/e2e/network/ensureloadbalancer.go]: could you use `utils.Logf` for logs?

Comment (nilo19) [reply tests/e2e/network/ensureloadbalancer.go]: Sure.

File: tests/e2e/network/ensureloadbalancer.go [hunk 2]
```diff
@@ -144,6 +145,12 @@ var _ = FDescribe("Ensure LoadBalancer", func() {
 		}()
 		By("Waiting for exposure of internal service with specific IP")
 		err = utils.WaitUpdateServiceExposure(cs, ns.Name, serviceName, ip1, true)
+		list, errList := cs.CoreV1().Events(ns.Name).List(metav1.ListOptions{})
+		Expect(errList).NotTo(HaveOccurred())
+		utils.Logf("Events list:")
+		for i, event := range list.Items {
+			utils.Logf(fmt.Sprintf("%d. %v", i, event))
```
Comment (justaugustus) [inline tests/e2e/network/ensureloadbalancer.go]:
```suggestion
			utils.Logf("%d. %v", i, event)
```

File: tests/e2e/network/ensureloadbalancer.go [hunk 3]
```diff
@@ -186,6 +193,12 @@ var _ = FDescribe("Ensure LoadBalancer", func() {
 
 		By("Waiting for exposure of the original service without assigned lb private IP")
 		ip1, err := utils.WaitServiceExposure(cs, ns.Name, serviceName)
+		list, errList := cs.CoreV1().Events(ns.Name).List(metav1.ListOptions{})
+		Expect(errList).NotTo(HaveOccurred())
+		utils.Logf("Events list:")
+		for i, event := range list.Items {
+			utils.Logf(fmt.Sprintf("%d. %v", i, event))
```
Comment (justaugustus) [inline tests/e2e/network/ensureloadbalancer.go]:
```suggestion
			utils.Logf("%d. %v", i, event)
```

File: tests/e2e/network/ensureloadbalancer.go (hunk 2 above)
Comment (nilo19) [reply tests/e2e/network/ensureloadbalancer.go]: Seems `fmt.Sprintf` is redundant. I'll amend it. Thanks.

**Comment by justaugustus** [comment]
@feiskyer -- as an aside, why are we using @acs-bot here?

File: tests/e2e/network/ensureloadbalancer.go (hunk 2 above)
Comment (feiskyer) [reply tests/e2e/network/ensureloadbalancer.go]: make sense. please remove the fmt

**Comment by feiskyer** [comment]
@justaugustus acs-bot is used to setup http://prow.upstream.azure-containers.io/. We're using this private prow to validate and troubleshoot some e2e tests.

**Comment by justaugustus** [comment]
@nilo19 -- you need to gofmt that file. The tests are failing because `fmt` is still in the import declaration and it is unused:

```
W0726 09:32:27.092] tests/e2e/network/ensureloadbalancer.go:20:2: imported and not used: "fmt"
```

**Comment by nilo19** [comment]
> @nilo19 -- you need to gofmt that file. The tests are failing because `fmt` is still in the import declaration and it is unused:
> 
> ```
> W0726 09:32:27.092] tests/e2e/network/ensureloadbalancer.go:20:2: imported and not used: "fmt"
> ```

Got it. Thank you very much.

**Pull request merged**